Add merge_GSEA function

DanielGarbozo opened the pull request:
I added the `merge_GSEA` function to obtain an output tsv file from all collections.
I change the documentation as well.

DanielGarbozo committed:
Add merge_GSEA.R function

DanielGarbozo committed:
Add documentation for merge_GSEA

danoguevara commented:
- Make sure the imported packages for your functions have the proper syntax within your function.
- You are missing adding the dependencies to the DESCRIPTION file. Do it by applying `usethis::use_package("pkg1")`.
- Make sure to automatically generate the `.Rd` file of your function with the `devtools::document()` package.
- Avoid parsing arguments into the function (`function.R <arg> <arg>`) like a CLI tool. R functions are better used when applied directly like this: `function(arg1, arg2)`.

DanielGarbozo committed:
Update merge_GSEA.R

Refactored merge_GSEA to use function arguments instead of CLI parsing

DanielGarbozo committed:
Add new function: plot_GSEA

DanielGarbozo committed:
Update NAMESPACE

Adds plot_global_GSEA to the NAMESPACE file so it can be accessed when the package is loaded.

DanielGarbozo committed:
Document plot_global_GSEA with roxygen2

Document plot_global_GSEA generated by roxygen2

danoguevara committed:
Merge branch 'main' into new_GSEA_functions
